src/domains/wallet-common/05_wallet_cdn.tf

```diff
@@ -0,0 +1,105 @@
+/**
+ * wallet resource group
+ **/
+resource "azurerm_resource_group" "wallet_fe_rg" {
+  name     = "${local.project}-fe-rg"
+  location = var.location
+
+  tags = var.tags
+}
+
+/**
+ * CDN
+ */
+module "wallet_fe_cdn" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=v6.15.2"
+
+  name                  = "fe"
+  prefix                = local.project
+  resource_group_name   = azurerm_resource_group.wallet_fe_rg.name
+  location              = var.location
+  hostname              = "${var.dns_zone_prefix}.${var.external_domain}"
+  https_rewrite_enabled = true
+
+  index_document     = "index.html"
+  error_404_document = "index.html"
+
+  dns_zone_name                = azurerm_dns_zone.wallet_public.name
+  dns_zone_resource_group_name = azurerm_dns_zone.wallet_public.resource_group_name
+
+  keyvault_resource_group_name = module.key_vault.resource_group_name
+  keyvault_subscription_id     = data.azurerm_subscription.current.subscription_id
+  keyvault_vault_name          = module.key_vault.name
+
+  querystring_caching_behaviour = "BypassCaching"
+
+  global_delivery_rule = {
+    cache_expiration_action       = []
+    cache_key_query_string_action = []
+    modify_request_header_action  = []
+
+    # HSTS
+    modify_response_header_action = [{
+      action = "Overwrite"
+      name   = "Strict-Transport-Security"
+      value  = "max-age=31536000"
+      },
+      # Content-Security-Policy
+      {
+        action = "Overwrite"
+        name   = var.env_short == "p" ? "Content-Security-Policy-Report-Only" : "Content-Security-Policy"
```

Comment:
Why `Content-Security-Policy-Report-Only` for prod env? I would set `Content-Security-Policy` as this product is a new one. What do you think?

Reply:
I left it as it is on the Checkout CDN for now given that their stack is similar